### PR TITLE
Added three flags s3-server, bucket and directory

### PR DESCRIPTION
**Pull request by Rajalakshmi-Girish**
Have added the flags s3-server, bucket and directory to enable the option of passing the S3 storage bucket details.

These details are used by the k8s-ansible submodule for the task of downloading the k8s bits while cluster creation.

### CODE REVIEW

**Comment by ltccci** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by: *<a href="https://github.com/ppc64le-cloud/kubetest2-plugins/pull/16#" title="Author self-approved">Rajalakshmi-Girish</a>*
To complete the [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process), please assign 
You can assign the PR to them by writing `/assign ` in a comment when ready.

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=ppc64le-cloud%2Fkubetest2-plugins).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/ppc64le-cloud/kubetest2-plugins/blob/master/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Comment by ltccci** [comment]
@Rajalakshmi-Girish: The following test **failed**, say `/retest` to rerun all failed tests:

Test name | Commit | Details | Rerun command
--- | --- | --- | ---
pull-kubetest2-plugins-terraform-validate | cda2f31a80050f4020b55856119ad0d71a72e574 | [link](https://prow.k8s-prow-c7f9fbfa2832409e2a3a6666e9dc52a3-0000.us-south.containers.appdomain.cloud/view/s3/prow-logs/pr-logs/pull/ppc64le-cloud_kubetest2-plugins/16/pull-kubetest2-plugins-terraform-validate/1313787353038327808) | `/test pull-kubetest2-plugins-terraform-validate`

[Full PR test history](https://prow.k8s-prow-c7f9fbfa2832409e2a3a6666e9dc52a3-0000.us-south.containers.appdomain.cloud/pr-history?org=ppc64le-cloud&repo=kubetest2-plugins&pr=16). [Your PR dashboard](https://prow.k8s-prow-c7f9fbfa2832409e2a3a6666e9dc52a3-0000.us-south.containers.appdomain.cloud/pr?query=is:pr+state:open+author:Rajalakshmi-Girish).

<details>

Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [kubernetes/test-infra](https://github.com/kubernetes/test-infra/issues/new?title=Prow%20issue:) repository. I understand the commands that are listed [here](https://go.k8s.io/bot-commands).
</details>

File: data/data/config.tf
Comment (mkumatag) [inline data/data/config.tf:14]: don't need these entries because we aren't using them in the TF scripts

File: pkg/tfvars/tfvars.go
Comment (mkumatag) [inline pkg/tfvars/tfvars.go:6]: see if we can `omitempty` entry to these variables so that when we don't mention these params, it will use the default ones.

**Review by mkumatag — CHANGES_REQUESTED** [review]

**Comment by mkumatag** [comment]
Fix the formatting as well: `go fmt ./...`

**Comment by ltccci** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by: *<a href="https://github.com/ppc64le-cloud/kubetest2-plugins/pull/16#issuecomment-704978794" title="Approved">mkumatag</a>*, *<a href="https://github.com/ppc64le-cloud/kubetest2-plugins/pull/16#" title="Author self-approved">Rajalakshmi-Girish</a>*
To complete the [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process), please assign 
You can assign the PR to them by writing `/assign ` in a comment when ready.

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=ppc64le-cloud%2Fkubetest2-plugins).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/ppc64le-cloud/kubetest2-plugins/blob/master/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Comment by ltccci** [comment]
[APPROVALNOTIFIER] This PR is **APPROVED**

This pull-request has been approved by: *<a href="https://github.com/ppc64le-cloud/kubetest2-plugins/pull/16#issuecomment-704980626" title="Approved">mkumatag</a>*, *<a href="https://github.com/ppc64le-cloud/kubetest2-plugins/pull/16#" title="Author self-approved">Rajalakshmi-Girish</a>*

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=ppc64le-cloud%2Fkubetest2-plugins).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details >
Needs approval from an approver in each of these files:

- ~~[OWNERS](https://github.com/ppc64le-cloud/kubetest2-plugins/blob/master/OWNERS)~~ [mkumatag]

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Pull request merged**
